packages/api-kit/src/types/safeTransactionServiceTypes.ts

```diff
@@ -325,3 +286,12 @@ export type AddSafeOperationProps = {
     validAfter?: number
   }
 }
+
+export type GetSafeOperationConfirmationListProps = {
```

Comment:
It is a little bit weird to use `limit` and `offset` as a string since they are associated with record count in a database. I imagine that the interface will be something like `apiKit.getSafeOperationConfirmations({ safeOperationHash, limit: '10' })`.

Reply:
Yes good point. Changed the type to number